misc/iceberg-combined-catalog/build.gradle.kts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2023 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+description = "catalog-lakehouse-iceberg"
```

Comment:
It'a a bit strange to have a new module for this feature, I think it is not so necessary.

Reply:
ok